<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageMiniLabUI</class>
 <widget class="QMainWindow" name="ImageMiniLabUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageMiniLab</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="LabCtrlWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>实验类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ExpTypeComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>实验图片：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ExpImgLineEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SelectImgPushButton">
           <property name="text">
            <string>选择</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>实验参数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="page"/>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="LoadTestDataPushButton">
           <property name="text">
            <string>载入测试数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="GoExpPushButton">
           <property name="text">
            <string>进行实验</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="ImgShowWidget" native="true">
      <widget class="QLabel" name="SrcImgShowLabel">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>11</y>
         <width>120</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SrcImgShowLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="DstImgShowLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>120</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DstImgShowLabel</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1333</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
